kpi/utils/object_permission_helper.py

```diff
@@ -78,7 +64,7 @@ def get_user_permission_assignments(cls, affected_object, user,
 
         if not user or user.is_anonymous:
             filtered_user_ids = [affected_object.owner_id]
-        elif not cls.user_can_share(affected_object, user):
+        elif not affected_object.has_perm(user, PERM_MANAGE_ASSET):
```

Comment:
So obvious ;-)

Reply:
I almost left `user_can_share()`, but I decided to follow through on my `TODO` :smile:

Reply:
Wise decision ;-)